src/site/content/en/fast/codelab-two-ways-to-prefetch/index.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: codelab
+title: "Two ways to prefetch: <link> tags and HTTP headers"
+authors:
+  - demianrenzulli
+description: |
+  In this codelab, learn how to improve the performance of a page by prefetching resources.
+date: 2019-00-04
+glitch: two-ways-to-prefetch
+---
+
+In this codelab, you'll implement prefetching in two ways: with `<link rel="prefetch">` and with HTTP `Link` header. 
+
+The sample app is a website that has a promotional landing page with a special discount for the shop's best selling t-shirt. Since the landing page links to a single product, it's safe to assume that a high percentage of users will navigate to the product details page. This makes the product page a great candidate to prefetch on the landing page.
+
+## Measure performance
+
+First establish the baseline performance:
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+{% Instruction 'devtools-network', 'ol' %}
+
+1. In the **Throttling** drop-down list, select **Fast 3G** to simulate a slow connection type.
+1. To load the product page, click **Buy now** in the sample app.
+
+The `product-details.html` page takes about 600 ms to load:
+
+<img class="w-screenshot" src="./network-panel-one.png" alt="Network panel showing load times for product-details.html">
+
+## Prefetch the product page with `<link rel="prefetch">`
+
+To improve navigation, insert a `prefetch` tag in the landing page to prefetch the `product-details.html` page:
+
+- Add the following `<link>` element to the head of the `views/index.html` file:
+
+```html/7
```

Comment:
Can you please switch to green code highlighting wherever you are adding new lines? I think yellow should be reserved for just highlight the existing code if you want to explain something. You can find instructions here https://web.dev/handbook/markup-code/#code-highlighting

Reply:
Actually, you only get the green when you're doing a delete-and-replace. If you're just highlighting a new line, yellow is your only option.

Reply:
Last time I tried it was possible, here's an example: https://web.dev/codelab-extract-and-inline-critical-css/#configure-critical

If that pattern is deprecated or if green vs. yellow highlighting is not important, feel free to close these comments.

Reply:
Verified that green highlighting is the way to go, here's the code snippet used in the example above:

`js/1-14/`

cc @mfriesenhahn to consider adding this pattern to the handbook.

Reply:
I thought the same as @mfriesenhahn, but turns out @mihajlija works! Adding this.

Reply:
Also: I've noticed that when highlighting JS, it colors the code inside the <script></script> tags, but not the tags. Wondering if this is intentional or a bug cc // @robdodson